flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "skip-arches": ["aarch64"]
```

Comment:
why?

Reply:
Because I have never tested this arch and I can't test it on my current setup. In doubt, I consider that was is not tested is not working.

Reply:
then it will never be tested.

if it builds it can be tested.

just remove the file

Reply:
Done!